amgut/lib/data_access/redcap.py

```diff
@@ -0,0 +1,115 @@
+from urllib import urlencode
+from json import loads
+
+from tornado.web import HTTPError
+from tornado.httpclient import HTTPClient
+
+from amgut.lib.config_manager import AMGUT_CONFIG
+from amgut.lib.data_access.sql_connection import TRN
+
+
+def _make_request(data):
+    body = urlencode(data)
+    client = HTTPClient()
+    response = client.fetch(AMGUT_CONFIG.redcap_url, method='POST',
+                            headers=None, body=body)
+    # Response not always JSON, so can't always use loads
```

Comment:
This comment does not reflect the code as a `loads` is used on line 18. Or is a valid response one that is not JSON?